libcxx/test/configs/apple-libc++-backdeployment.cfg.in

```diff
@@ -44,12 +44,8 @@ BACKDEPLOYMENT_PARAMETERS = [
 config.substitutions.append(('%{flags}',
     '-isysroot {}'.format('@CMAKE_OSX_SYSROOT@') if '@CMAKE_OSX_SYSROOT@' else ''
 ))
-config.substitutions.append(('%{compile_flags}',
-    '-nostdinc++ -I %{include} -I %{libcxx}/test/support'
-))
-config.substitutions.append(('%{link_flags}',
-    '-nostdlib++ -L %{lib} -lc++'
-))
+config.substitutions.append(('%{compile_flags}', config._common_compile_flags_no_target))
```

Comment:
Is there any reason why we are not just adding `@CMAKE_CXX_FLAGS_INIT@` to the front of all of these configs? The intent was to communicate between `cmake-bridge.cfg.in` and the various configs only via Lit substitutions, and I think that is a useful property to retain. I'd like to avoid passing arbitrary information down to config files via the `config` object.

Reply:
Sure I can revert the current changes, I just thought it might be good to avoid duplicating the flags where possible. Will update to only add the CMAKE variables.

Reply:
Updated to just add the extra arg.